### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by jgigault**
Note that the images should be visible once the PR is merged.

TODO:
- [x] add script of our presentation including images
- [x] add links to reference
- [ ] write a conclusion
- [x] add link to youtube video

TIP:
- To get a preview of the README, click here: https://github.com/perangusta/one-day-in-rails6-world/tree/documentation/readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,119 @@
...
+![Image](One Day In Rails 6 World_Page_02.png)
 
-* Configuration
+## [`#cb20c48`](https://github.com/perangusta/one-day-in-rails6-world/commit/cb20c48) - Rails new
 
-* Database creation
+Today, we will be creating a new application to publish our changelogs from Per Angusta's product.
 
-* Database initialization
+As former Rails developers, we run the `rails new` command to bootstrap our new project. We are able to see the appearing of three brand new dependencies (`zeitwerk`, `actionmailbox`, `actiontext`) and a new folder in the tree `app/javascripts/packs` which is marking the arrival of Webpack as the main JavaScript dependencies manager (via `webpacker`).
```
Comment (francois-belle) [inline README.md]: I think `former` means `ancien` in this context.

File: README.md [hunk 2]
```diff
@@ -1,24 +1,119 @@
...
+
+## [`#013d262`](https://github.com/perangusta/one-day-in-rails6-world/commit/013d262) - Add rich text to Changelog
+
+Then the setup of `Actiontext` is as easy as a few lines of code (Model and views), but hey, I have not migrated anything! Amazing...
+
+![Image](One Day In Rails 6 World_Page_05.png)
+
+## [`#4006f5c`](https://github.com/perangusta/one-day-in-rails6-world/commit/4006f5c) - Avoid N+1 query issue
+
+Did you know that rich content is stored in a third party table `action_text_rich_texts`? Let's avoid the N+1 query issue in our index view by using the builtin helpers for eager loading.
```
Comment (francois-belle) [inline README.md]:
```
Correctif, le helper `.with_rich_text_content_and_embeds` ne solutionne pas le problème de requête N+1 et vont être complètement supprimés.
https://github.com/rails/rails/issues/36177
```

File: README.md [hunk 3]
```diff
@@ -1,24 +1,119 @@
...
+
+A changelog without any title would be kind of strange, is'nt it? Let's add a `presence` validation and our first test. We can now take advantage of the new `kind_of?` interface to make this kind of tests more readable.
+
+![Image](One Day In Rails 6 World_Page_08.png)
+
+## [`#5ea2a86`](https://github.com/perangusta/one-day-in-rails6-world/commit/5ea2a86) - Parallel Testing
+
+We should also add a validation on the description field because users always need to be guided. Let's say the description must be 3 characters length or more, and add a test.
+
+Did you know that Rails 6 is now bringing parallel testing by default? By appending a `sleep` command, we can highlight the new bahevior: Before around `8 seconds` and after around `5.5 seconds` using parallel testing.
```
Comment (francois-belle) [inline README.md]: `behavior`

File: README.md [hunk 4]
```diff
@@ -1,24 +1,119 @@
...
+
+Be aware that it may not be compatible with your existing test suite!
+
+![Image](One Day In Rails 6 World_Page_09.png)
+
+## [`#5886e94`](https://github.com/perangusta/one-day-in-rails6-world/commit/5886e94) - ActiveSupport::ActionableError
+
+What about not to publish our changelogs by default on creation by implementing  a status with an `enum`. Let's create the migration, update the model, views and controller and manually test it!
```
Comment (francois-belle) [inline README.md]: I would say
```
What about not publishing...
```

`implementing  a status` -> extra space

File: README.md (hunk 2 above)
Comment (jgigault) [reply README.md]: I have added an ERRATUM

**Pull request merged**